crates/next-core/src/next_image/mod.rs

```diff
@@ -65,11 +65,7 @@ impl ContentSource for NextImageContentSource {
         // TODO: consume the assets, resize and reduce quality, re-encode into next-gen
         // formats.
         if let Some(path) = url.strip_prefix('/') {
-            let asset = this.asset_source.get(path, Default::default());
-            let inner = asset.await?;
-            if let ContentSourceResult::Result { .. } = &*inner {
-                return Ok(asset);
-            }
+            return Ok(this.asset_source.get(path, Default::default()));
```

Comment:
I can't think of a case where we really want to fall into the `HttpProxy` redirect below.